views/show.ejs

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/public/style.css">
+    <title><%= resi.basics.name%>'s Resume</title>
+</head>
+<body>
+    <div>
+
+        <div><img src="<%= resi.basics.picture %>" alt="<%= resi.basics.name %>"></div>
+
+        <div> 
+            <h1><%= resi.basics.name %>: <%= resi.basics.label %></h1>
+        </div> 
+
+        <details>
+            <summary>
+                About <%=resi.basics.name%>
+            </summary>
+            <p>
+                <%= resi.basics.summary %>
+            </p>
+        </details>
+        <div>
+           <h3>Where to find me:</h3> 
+        <% for(property in resi.basics.location){%>
+            <%=resi.basics.location[property]%>
+            <%}%>
+        </div>
+        <div>
```

Comment:
You could have nested some of these divs to improve the structure of the page, and make styling the page easier